Samples/Swift/nhubsample/nhubsample/NotificationDetailViewController.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  Copyright © 2018 Microsoft All rights reserved.
+//  Licensed under the Apache License (2.0).
+//
+
+import UIKit
+
+class NotificationDetailViewController: UIViewController {
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var dismissButton: UIButton!
+    @IBOutlet weak var bodyLabel: UILabel!
+    
+    var _userInfo: [String: Any]?
```

Comment:
Use of Any is bad. Can you come up with a `Codable` object that matches what you want? I know this would affect the Objective-C version, but this seems bad.

Reply:
We basically want a dictionary type .. that's what is deserialized from the notification content. Is there a standard Swift type for a JSON dictionary?

Reply:
We can fix this in the Objective-C version as well, depending on the approach we use here.